keadm/cmd/keadm/app/cmd/util/cloudinstaller.go

```diff
@@ -181,6 +181,8 @@ func (cu *KubeCloudInstTool) TearDown() error {
 
 	//Kill cloudcore process
 	cu.KillKubeEdgeBinary(KubeCloudBinaryName)
+	// clean kubeedge namespace
+	cu.cleanNameSpace("kubeedge", cu.KubeConfig)
```

Comment:
Could we return the error here? And also return in https://github.com/kubeedge/kubeedge/blob/master/keadm/cmd/keadm/app/cmd/reset.go#L88

Reply:
@fisherxu  Done, thanks for reminding.